sdk/communication/communication-phone-numbers/karma.conf.js

```diff
@@ -63,6 +69,9 @@ module.exports = function (config) {
       "AZURE_CLIENT_SECRET",
       "AZURE_TENANT_ID",
       "COMMUNICATION_SKIP_INT_PHONENUMBERS_TESTS",
+      "INCLUDE_UPDATE_CAPABILITIES_LIVE_TESTS",
```

Comment:
In some SDKs this is set as `SKIP_UPDATE_CAPABILITIES_LIVE_TESTS`, which means a `true` value will not execute them. Here, a `true` value **will** execute them. I believe we should standardize it.

Reply:
I agree, I will make the change accordingly. The only reason to do it this way was that that it was similar to the `INCLUDE_PHONENUMBER_LIVE_TESTS` variable; but it is more valuable to have consistency between our SDKs.

Reply:
This was updated, can you please review @lucasrsant ?